### PR TITLE
feature: compatibility with sonarqube 9.0

### PR DESCRIPTION
**Pull request by rpaasche**

### CODE REVIEW

**Comment by deepy** [comment]
Great work! This looks fine, but I can't test it until tomorrow, based on #42 this sounds like it'd work for 8.9 and with 8.9 being the current LTS that'd be a great target

**Comment by rpaasche** [comment]
Hmmm strange. 2.1.3 worked just fine on 8.9.43852 for us.

**Comment by ccmuhub** [comment]
I've just tested this PR on sonarqube-enterprise-9.0.0.45539.zip to Crowd 4.3.0 and can confirm that it is working as expected (9.0.0 with the current 2.3.0 plugin version did not work). Gets my tick of approval! Thanks rpaasche

File: src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java [hunk 1]
```diff
@@ -94,7 +96,12 @@ public CrowdGroupsProvider(CrowdClient crowdClient) {
     return ImmutableList.copyOf(groups);
   }
 
-  @Override
```
Comment (deepy) [inline src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java:97]: Shouldn't this `@Override` be on the groups provider on the sonar 9.0 new method?

**Review by deepy — COMMENTED** [review]
Having looked at this it looks good, I think there's a missing `@Override` and it looks like this should build fine against `8.9.0.43852` as well (which is the current LTS) and that should work with 9.0 as well

File: src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java (hunk 1 above)
Comment (rpaasche) [reply src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java:97]: Will switch to 8.9 next week and check the overrides.

**Comment by rpaasche** [comment]
@deepy do mind switching to java 11 since the server dose not support running on java 8 anymore?

https://docs.sonarqube.org/8.9/requirements/requirements/

File: src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java (hunk 1 above)
Comment (deepy) [reply src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java:97]: I've just tested with 8.9 in the pom and with the overrides, it works on SQ 9.0, I'll test on actual 8.9 and check that the groups are synced tomorrow or when the icecream cools me down

**Comment by deepy** [comment]
no complains about 11 from here, I'll test that as well to make sure there's nothing sneaking up on us

**Comment by deepy** [comment]
no complains about 11 from here, I'll test that as well to make sure there's nothing sneaking up on us

**Comment by deepy** [comment]
I wonder if I can automate the integration testing from my end, starting two servers and doing some REST calls shouldn't be too difficult

**Comment by deepy** [comment]
Alright I tested this with `8.9` and group syncing works so I think it should be safe to release

**Pull request merged**